src/metric_config.cpp

```diff
@@ -66,7 +66,15 @@ Status MetricConfig::parseMetricsConfig(const rapidjson::Value& metrics, bool fo
     if (v.HasMember("metrics_list")) {
         status = parseMetricsArray(v["metrics_list"]);
     } else {
-        setDefaultMetricsTo(metricsEnabled);
+        setDefaultMetricsTo(this->metricsEnabled);
+    }
+
+    if (status == StatusCode::OK && this->metricsEnabled) {
+        SPDLOG_LOGGER_INFO(modelmanager_logger, "Metrics enabled.");
+        SPDLOG_LOGGER_DEBUG(modelmanager_logger, "Enabled metrics list: ");
```

Comment:
It could be better to concat all metrics into one string with no new lines using stringstream